packages/hash/api/src/seed-data/index.ts

```diff
@@ -0,0 +1,104 @@
+import { Logger } from "@hashintel/hash-backend-utils/logger";
+import { GraphApi } from "@hashintel/hash-graph-client";
+import { OrgModel, OrgSize } from "../model";
+import { workspaceAccountId } from "../model/util";
+import { ensureDevUsersAreSeeded } from "./dev-users";
+import { PageDefinition, seedPages } from "./seed-pages";
+
+// Seed Org with some pages.
+const seedOrg = async (params: {
+  graphApi: GraphApi;
+  logger: Logger;
+}): Promise<OrgModel> => {
+  const { graphApi, logger } = params;
+
+  const sharedOrgModel = await OrgModel.createOrg(graphApi, {
+    name: "HASH",
+    shortname: "hash",
```

Comment:
Is there a reason we're not using `WORKSPACE_ACCOUNT_NAME` and `WORKSPACE_ACCOUNT_SHORTNAME` as we did previously?

Reply:
Thanks, good point! a4632932ca9205874fd7142e2d0a106d1e15d0f5